pkg/compiler/vars.go

```diff
@@ -12,9 +14,10 @@ type varScope struct {
 }
 
 type varContext struct {
-	importMap map[string]string
-	expr      ast.Expr
-	scope     []map[string]varInfo
+	importMap    map[string]string
+	expr         ast.Expr
+	assertToType *stackitem.Type
```

Comment:
`castTo`?

Reply:
This code is removed, see https://github.com/nspcc-dev/neo-go/pull/2720#discussion_r984290723.